src/js/modules/infragistics.ui.combo.js

```diff
@@ -32,6 +32,7 @@
 			"jquery-ui",
 			"./infragistics.util",
```

Comment:
I'll also remove jquery and jquery-ui, since they are dependencies for widget as well.